### ISSUE TITLE
Check and update Twitter handles for councils

### ISSUE THREAD

**Issue by zarino** [mysociety/caps#215]
On a call this morning, Annie mentioned she’d recently checked through the LGA’s list of Twitter handles for councils, and spotted a few were wrong.

We should check through our copy of the list, and make any amendments required. Ideally before the Scorecards go live (but it’s not a deal-breaker.) Annie might be able to help to do this.

**Comment by zarino** [comment]
Update: it looks like we pull the twitter handles for English and Welsh councils straight from the LGA website – https://github.com/mysociety/caps/blob/8ae60472efcfa9b9421b6fc62d24aa873efa4cd8/caps/management/commands/add_council_websites.py#L33-L62 – so maybe we should be asking the LGA to update their data ;-)

**Comment by struan** [comment]
Yeah, we get the data from the LGA. We could probably source this from wikipedia as we do with some of the other ones and then we'd at least be able to update it.

**Comment by struan** [comment]
ok, looks like wikidata is a lot less complete than the LGA page so let's not use that.

**Comment by zarino** [comment]
But we could _add_ the data _from_ the LGA page _to_ WikiData, and then get the data back _from_ WikiData, right? Make the world a better place, and all that?

**Comment by struan** [comment]
I mean, yes, but not right now.
